packages/components/src/Tooltip/Tooltip.stories.tsx

```diff
@@ -27,56 +20,62 @@ const meta = {
       description:
         'The additional offset across the cross acis from the trigger element',
     },
-    delay: {
-      control: {
-        type: 'number',
-      },
-      description: 'The delay before the tooltip is shown',
-    },
-    trigger: {
-      control: {
-        type: 'select',
-      },
-      options: [undefined, 'focus'],
-      defaultValue: undefined,
-      description: 'The trigger type (default = focus AND hover)',
-    },
     placement: {
       control: {
         type: 'select',
       },
       options: ['top', 'bottom', 'left', 'right'],
       description: 'The placement of the tooltip',
     },
+    shouldFlip: {
+      control: {
+        type: 'boolean',
+      },
+      description: 'Should the tooltip be automatically be flipped',
+    },
     containerPadding: {
       control: {
         type: 'number',
       },
-      description: 'The padding around the tooltip',
+      description:
+        'The placement padding that should be applied between the element and its surrounding container.',
     },
-    shouldFlip: {
+    open: {
```

Comment:
The open and isOpen functions do not seem to work properly. If I understand the documentation correctly, it must also be with controlled state?!

https://react-spectrum.adobe.com/react-aria/Tooltip.html#controlled-open-state